test/browser_tests/utils/mixin.js

```diff
@@ -0,0 +1,35 @@
+/* ==========================================================================
+   Mixin
+
+   Code copied from the following with moderate modifications :
+
+   - https://github.com/maslennikov/shallow-extend
+     Copyright (c) 2014 Alexey Maslennikov
+   ========================================================================== */
+
+'use strict';
+
+/**
+* Copies properties of all sources to the destination object overriding its own
+* existing properties. When assigning from multiple sources, fields of every
+* next source will override same named fields of previous sources.
+*
+* @param {object} destination object.
+* @returns {object} assigned destination object.
+*/
+function mixin( destination ) {
+  destination = destination || {};
+
+  for ( var i = 1; i < arguments.length; i++ ) {
```

Comment:
So I've deleted this file because we are already including `lodash` as a dev dependency.